Bindings/Python/setup.py

```diff
@@ -17,6 +31,7 @@
       url='http://opensim.stanford.edu/',
       license='Apache 2.0',
       packages=['opensim'],
+      data_files = [('Scripts', bin_files)],
```

Comment:
This line will copy all files in the bin folder to the Python environment Scripts folder in pip installation. In conda installation however, the `bin_files` in empty because the package have all the `dll` and `exe` files and will put them into `Library/bin`.